benchmarks/backend/config/setup.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
```

Comment:
Setup of backend?

Reply:
General setup of backend and database. Described in doc.